Logging/LogHandler/FileLogHandler.cs

```diff
@@ -151,45 +114,28 @@ private void CreateWriter()
             };
         }
 
-        protected override void DisposeSelf()
+        public void Dispose()
         {
             m_Writer.Dispose();
         }
 
-        public void HandleLog(
-            LogLevel level,
-            string message,
-            string callerDerivedTypeName,
-            string callerPath,
-            string callerName,
-            int callerLine)
+        protected override void HandleFormattedLog(LogLevel level, string formattedLog)
         {
             if ((int)level < (int)MinimumLevel)
             {
                 return;
             }
```

Comment:
If we do keep `AbstractLogHandler` we should drop this down into the type. No sense in doing all the work to format a log only to bail out. This is a useful concept for all log handlers.

Reply:
I was on the fence about that, since setting a minimum log level seems specific to file writing - you definitely want the option to see only errors in a big log file, but you'd never prevent debug logs/warnings from being sent to any console/editor/IDE would you?

Reply:
For filtering purposes potentially. Unity's console has that filtering functionality baked in but it filters from all logs.

I know we had a custom log handler in our console application for a bit that only showed a subset of messages to the user. I ended up simplifying and just monitoring the log file directly then parsing out the log level from the message but that was more out of laziness than implementing the ideal approach. A custom handler was probably the best way to go if I had more time.

I could also see restricting log levels in release builds vs debug builds for performance reasons or to prevent logging sensitive data.